ts/screens/modal/components/IdentificationNumberPad.tsx

```diff
@@ -82,26 +82,19 @@ export const IdentificationNumberPad = (
           onValidate={onPinValidated}
         />
       </View>
-      <VSpacer size={48} />
-      <View>
-        <NumberPad
-          value={value}
-          deleteAccessibilityLabel={I18n.t("global.buttons.delete")}
-          onValueChange={onValueChange}
-          variant={numberPadVariant}
-          {...biometricsConfig}
-        />
-      </View>
       {isDevEnv && (
         <View
           accessible={false}
           importantForAccessibility="no-hide-descendants"
           accessibilityElementsHidden
-          style={IOStyles.alignCenter}
+          style={{
+            alignSelf: "center",
+            position: "absolute",
+            bottom: -56
+          }}
```

Comment:
Why do we use an inline style instead of a style object?

Reply:
Technically, it's already a style object, just not managed with the `StyleSheet` method. There is no significant performance improvement between the two solutions, and I don't think the `StyleSheet` style object improves the readability of the code. In fact, I think it's the opposite in this particular case.